Formula/objconv.rb

```diff
@@ -0,0 +1,23 @@
+class Objconv < Formula
+  desc "Object file converter and disassembler"
+  homepage "http://www.agner.org/optimize/#objconv"
+  url "http://www.agner.org/optimize/objconv.zip"
```

Comment:
Unfortunately, the maintainer of this software doesn't provide a versioned link. There is a GitHub mirror https://github.com/vertis/objconv, which would allow checking out a specific revision. Would that be acceptable?